data_ingestion/common/publisher.py

```diff
@@ -0,0 +1,25 @@
+from google.cloud import pubsub_v1
```

Comment:
I'm not familiar with cloud lingo, but is publisher a descriptive enough name for this file?

Also is this function going to be called multiple places or just main.py? If it was just the one place, I may be tempted to put it in main.py? Again, I'm not super familiar with cloud or python conventions. I just personally get confused when there are lots of vaguely named files haha :)

Reply:
I was thinking that this could be a place to add other logic for publishing messages, so I named it somewhat generically. I'm not opposed to another name though if you have suggestions.

As for putting it in main.py, I think it's probably best to avoid putting much logic in there since I think that would make it more likely for us to have conflicting changes when collaborating. Also main.py is sort of by definition vague, so putting most logic in other files will help organize it I think.

Reply:
If we're sticking with Python, we should probably use this: https://www.python.org/dev/peps/pep-0008/#naming-conventions

Reply:
Was there a specific part of that URL you thought was pertinent to naming this file, Brian?

I think even just changing this to "pubsub_publisher" would personally help me with readability :)

Reply:
It's pretty light on details about naming modules:

"Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. Python packages should also have short, all-lowercase names, although the use of underscores is discouraged."

Reply:
Oops didn't see your comment when I made my comment. I'll change to pubsub_publisher